### DISCUSSION TITLE
Easy Starstruck

### DISCUSSION THREAD

**Discussion by miladkhalighi** [Schweinepriester/github-profile-achievements#388] [General]
Dont be shy 
Star my repo
https://github.com/miladkhalighi/Food-Delivery-Flutter-app
And i star yours ⭐

**Comment by longquoc47krb** [comment]


**Reply by miladkhalighi** [reply]
For sure

**Reply by WHAHA-HA** [reply]
@longquoc47krb 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by aarongeo1** [reply]
done https://github.com/aarongeo1/NUM-CLASSIFY

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Reply by miladkhalighi** [reply]
@danielr1996 done

**Comment by Jhonatanmarquess** [comment]
done!

**Reply by kshitij-halankar** [reply]
@Jhonatanmarquess please start my repo 
https://github.com/kshitij-halankar/TravelR

**Comment by WHAHA-HA** [comment]
@miladkhalighi 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by miladkhalighi** [reply]
@WHAHA-HA already done

**Comment by gogson** [comment]
@miladkhalighi done ! https://github.com/gogson/MidjourneyAssistant

**Reply by miladkhalighi** [reply]
@gogson done as well

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by HHNebula** [comment]
Done!
https://github.com/HHNebula/HHNebula

**Reply by miladkhalighi** [reply]
@HHNebula done

**Comment by idontknowtoobrother** [comment]
> Dont be shy Star my repo https://github.com/miladkhalighi/Food-Delivery-Flutter-app And i star yours ⭐

Already

**Reply by idontknowtoobrother** [reply]
https://github.com/idontknowtoobrother/WordMissingGame

**Reply by miladkhalighi** [reply]
@idontknowtoobrother done

**Comment by 41pha1** [comment]
Done
https://github.com/41pha1/MIDI-Converter

**Reply by miladkhalighi** [reply]
@41pha1 done as well

**Comment by drk1rd** [comment]
Done!
https://github.com/drk1rd/drk1rd

**Reply by miladkhalighi** [reply]
@drk1rd thanks, done too
